systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -189,4 +191,17 @@ public static void waitForKafkaTopicSpecStability(String topicName, String podNa
             return false;
         });
     }
+
+    public static List<KafkaTopic> getAllKafkaTopicsWithPrefix(String namespace, String prefix) {
+        return KafkaTopicResource.kafkaTopicClient().inNamespace(namespace).list().getItems()
+            .stream().filter(p -> p.getMetadata().getName().startsWith(prefix))
+            .collect(Collectors.toList());
+    }
+
+    public static void deleteAllKafkaTopicsWithPrefix(String namespace, String prefix) {
+        List<KafkaTopic> ktl = KafkaTopicUtils.getAllKafkaTopicsWithPrefix(namespace, prefix);
+        for (KafkaTopic kt : ktl) {
+            cmdKubeClient().namespace(namespace).deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
```

Comment:
Maybe you can simplify it like this. Also, do we want some wait for deletion of topics?
```suggestion
        KafkaTopicUtils.getAllKafkaTopicsWithPrefix(namespace, prefix).stream().forEach(topic -> cmdKubeClient().namespace(namespace).deleteByName(KafkaTopic.RESOURCE_SINGULAR, topic.getMetadata().getName()))
        }
```

Reply:
Interesting that you found this code in here, as I already use 
```
public static void deleteAllKafkaTopicsWithPrefix(String namespace, String prefix) {
        List<KafkaTopic> ktl = KafkaTopicUtils.getAllKafkaTopicsWithPrefix(namespace, prefix);
        for (KafkaTopic kt : ktl) {
            cmdKubeClient().namespace(namespace).deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
        }
    }
```